components/chassis.py

```diff
@@ -17,10 +19,40 @@ class Chassis:
     drive_motor_d = CANTalon
     sd = NetworkTable
 
+    # some calculatins that provide numbers used by the motion profiling
+    wheel_circumference = 6*0.0254*math.pi # m
```

Comment:
Ah, right. Good to know. Maybe that should also be a named constant. (If I asked, imagine how many of the students reading your code might ask.)